olm/prepare-community-operators-update.sh

```diff
@@ -25,23 +25,28 @@ do
   cd "${packageBaseFolderPath}"
 
   packageFolderPath="${packageBaseFolderPath}/deploy/olm-catalog/${packageName}"
+  sourcePackageFilePath="${packageFolderPath}/${packageName}.package.yaml"
   communityOperatorsLocalGitFolder="${packageBaseFolderPath}/generated/community-operators"
+  lastPackagePreReleaseVersion=$(yq -r '.channels[] | select(.name == "stable") | .currentCSV' "${sourcePackageFilePath}" | sed -e "s/${packageName}.v//")
 
   echo "   - Cloning the 'community-operators' GitHub repository to temporary folder: ${communityOperatorsLocalGitFolder}"
   
   rm -Rf "${communityOperatorsLocalGitFolder}"
   mkdir -p "${communityOperatorsLocalGitFolder}"
-  git clone https://github.com/operator-framework/community-operators.git "${communityOperatorsLocalGitFolder}" 2>&1 | sed -e 's/^/      /'
-
+  git clone https://github.com/che-incubator/community-operators.git "${communityOperatorsLocalGitFolder}" 2>&1 | sed -e 's/^/      /'
```

Comment:
why use a fork of the operator-framework repo?

Reply:
in order to push back to it and be able to open a PR against the upstream repo.
But you're right, since, later, I push to a given URL, I might have avoided this step.

Reply:
so... leave it? fix it?

Reply:
keept it ? (I'm lazy there) ;-)